### DISCUSSION TITLE
Menu component

### DISCUSSION THREAD

**Discussion by ecatugy** [mantinedev/mantine#3999] [Feedback]
Do you happen to have any updates on the release date for the menu generation component? I'm eagerly anticipating its availability for use in my projects.

**Comment by rtivital** [comment]
What is menu generation component?

**Reply by ecatugy** [reply]
An example is Antd, if Mantine had menu components like theirs, they would definitely be the best in the market.

[https://ant.design/components/menu](url)

**Reply by rtivital** [reply]
https://mantine.dev/core/nav-link/

**Reply by ecatugy** [reply]
> https://mantine.dev/core/nav-link/

This example is extremely basic, outside the reality of the market, it should have the effects of Collapsed inline menu, Vertical menu with inline submenus, dynamic switching mode (between inline and vertical).
